tests/testVisitInfo.py

```diff
@@ -33,10 +33,10 @@
 import lsst.afw.image as afwImage
 
 RotTypeEnumNameDict = {
-    afwImage.RotType_UNKNOWN: "UNKNOWN",
-    afwImage.RotType_SKY: "SKY",
-    afwImage.RotType_HORIZON: "HORIZON",
-    afwImage.RotType_MOUNT: "MOUNT",
+    afwImage.RotType.UNKNOWN: "UNKNOWN",
+    afwImage.RotType.SKY: "SKY",
+    afwImage.RotType.HORIZON: "HORIZON",
+    afwImage.RotType.MOUNT: "MOUNT",
```

Comment:
I would keep it too. But it might cause some problems later on. Lets keep it for now. We can always add back the old syntax as aliasses if needed.